daemon/config.cfg

```diff
@@ -9,6 +9,9 @@ database=xmusic
 file=xmusic-daemon.log
 level=DEBUG
 
+[YOUTUBE]
+key=AIzaSyC02GiHCafXnwCjYN7e7djR9ogys0aEGlE
+
```

Comment:
Thanks for reminding the issue of security.

I'll remove the configure session and delete the key on [Google API Console](https://console.developers.google.com). And now, it can apply the `key` which acquired from [Google API Console](https://console.developers.google.com) to the environment variable(`YOUTUBE_API_KEY`), it's can configure more easily when set up the docker service.